irohad/synchronizer/impl/synchronizer_impl.cpp

```diff
@@ -158,9 +158,9 @@ namespace iroha {
     void SynchronizerImpl::processNext(const consensus::PairValid &msg) {
       log_->info("at handleNext");
       const auto notify =
-          [this,
-           &msg](std::shared_ptr<const iroha::LedgerState> &&ledger_state) {
-            this->notifier_.get_subscriber().on_next(
+          [&msg](std::shared_ptr<const iroha::LedgerState> &&ledger_state) {
+            getSubscription()->notify(
+                EventTypes::kOnSynchronization,
```

Comment:
Why this is so? There were no integration yet. #824 is not merged. You just hide event from rxcpp - it can break some mechanics.

Reply:
Oo